brave/brave-core
# fix(wallet): Import with PrivateKey in Panel

## Description 
Fixes importing accounts with a `PrivateKey` in the Wallet `Panel`


Resolves <https://github.com/brave/brave-browser/issues/32478>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Open the Wallet `Panel` and navigate to the `Accounts` tab
2. Open the `More Menu` and click on `Import account`
3. Select `Ethereum` and then try to import an account with a `PrivateKey`, it should work.
4. Follow these same steps for `Solana` and `Filecoin` accounts.


Before:

https://github.com/brave/brave-core/assets/40611140/92103d0a-6140-4088-b5bb-0cffbbee9940

After:

https://github.com/brave/brave-core/assets/40611140/5cd8fa99-0ef1-4f0a-8b28-57dd4a2fe893
